### DISCUSSION TITLE
Refused to connect; during edit, add, view, delete.

### DISCUSSION THREAD

**Discussion by himanshu-al-we** [fabiocaccamo/django-admin-interface#245] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55629547/213871744-c4d362ca-bfb0-4017-90d9-cf047e173b08.png)

For all the operations highlighted in above screenshot, I'm getting "refused to connect" error (only while using admin-interface, works fine with default one).
![image](https://user-images.githubusercontent.com/55629547/213871691-ead1b123-18b8-4047-80cf-cc3f8bcc4d06.png)

Any resolution for this issue?

**Comment by fabiocaccamo** [comment, marked as answer]
I'm pretty sure you forgot to add `X_FRAME_OPTIONS = "SAMEORIGIN"` setting.

**Reply by himanshu-al-we** [reply]
Yea.. I missed settings snippet from documentation.
Thanks.
